### PR TITLE
UI :- Added option to remove Owner and Tier

### LINKED ISSUE #7473: Allow Removing The Owner After Assignment
**Feature**
Allow removing the owner after assignment under the `manage` tab.

**Describe the task**
Currently when we go to the `manage` tab for a table and assign an owner we are not able to unassign it back to nothing. Can you please add the capability to do so.

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes :

- Issue : Closes  #7473 
- Added option to remove owner
- Added option to remove Tier
- Select owner will be on top of list

#
### Type of change :

- [x] Improvement
- [x] New feature

#
### Frontend Preview (Screenshots) :
image

image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui

### CODE REVIEW

**Comment by harshach** [comment]
@Ashish8689 while we are on this PR can we add an option to remove Tier as well

**Comment by Ashish8689** [comment]
@harshach  ok, will add a option for Tier too!

File: openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/TeamDetailsV1.tsx [hunk 1]
```diff
@@ -420,6 +420,19 @@ const TeamDetailsV1 = ({
...
+    if (currentTeam) {
+      const updatedData: Team = {
+        ...currentTeam,
+        owner: undefined,
+      };
+
+      return updateTeamHandler(updatedData);
+    }
+
+    return Promise.reject();
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/TeamDetailsV1.tsx:433]: Why we need this?

File: openmetadata-ui/src/main/resources/ui/src/components/common/TierCard/TierCard.tsx [hunk 2]
```diff
@@ -95,7 +101,25 @@ const TierCard = ({ currentTier, hideTier, updateTier }: TierCardProps) => {
           paddingLeft: '16px',
           paddingTop: '12px',
         }}
-        title={<Typography.Title level={5}>Edit Tier</Typography.Title>}>
+        title={
+          <Row className="justify-between items-center">
+            <Typography.Title className="m-b-0" level={5}>
+              Edit Tier
+            </Typography.Title>
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/common/TierCard/TierCard.tsx:108]: Row must have Col as children

File: openmetadata-ui/src/main/resources/ui/src/pages/DatabaseSchemaPage/DatabaseSchemaPage.component.tsx [hunk 3]
```diff
@@ -411,6 +411,34 @@ const DatabaseSchemaPage: FunctionComponent = () => {
...
+      ...databaseSchema,
+      owner: undefined,
+    };
+
+    return new Promise<void>((_, reject) => {
+      saveUpdatedDatabaseSchemaData(updatedData as DatabaseSchema)
+        .then((res) => {
+          if (res) {
+            setDatabaseSchema(res);
+            reject();
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/DatabaseSchemaPage/DatabaseSchemaPage.component.tsx:425]: It should be resolved right?

File: openmetadata-ui/src/main/resources/ui/src/pages/database-details/index.tsx [hunk 4]
```diff
@@ -439,6 +439,34 @@ const DatabaseDetails: FunctionComponent = () => {
...
+      ...database,
+      owner: undefined,
+    };
+
+    return new Promise<void>((_, reject) => {
+      saveUpdatedDatabaseData(updatedData as Database)
+        .then((res) => {
+          if (res) {
+            setDatabase(res);
+            reject();
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/database-details/index.tsx:453]: `resolve()`??

File: openmetadata-ui/src/main/resources/ui/src/pages/service/index.tsx [hunk 5]
```diff
@@ -828,6 +829,38 @@ const ServicePage: FunctionComponent = () => {
...
+      owner: undefined,
+    } as unknown as ServiceOption;
+
+    return new Promise<void>((resolve, reject) => {
+      updateService(serviceName, serviceDetails?.id ?? '', updatedData)
+        .then((res) => {
+          if (res) {
+            setServiceDetails(res);
+
+            return resolve();
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/service/index.tsx:844]: I don't think we need to return just `resolve()` will work

File: openmetadata-ui/src/main/resources/ui/src/pages/service/index.tsx [hunk 6]
```diff
@@ -828,6 +829,38 @@ const ServicePage: FunctionComponent = () => {
...
+          } else {
+            showErrorToast(
+              jsonData['api-error-messages']['update-owner-error']
+            );
+          }
+
+          return reject();
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/service/index.tsx:851]: same as above

File: openmetadata-ui/src/main/resources/ui/src/pages/service/index.tsx [hunk 7]
```diff
@@ -828,6 +829,38 @@ const ServicePage: FunctionComponent = () => {
...
+        })
+        .catch((error: AxiosError) => {
+          showErrorToast(
+            error,
+            jsonData['api-error-messages']['update-owner-error']
+          );
+
+          return reject();
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/service/index.tsx:859]: same as above

**Review by chirag-madlani — CHANGES_REQUESTED** [review]

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**